From 79ceb26cfcf3f10ef824c72fc935be57826de768 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@st.com>
Date: Thu, 13 Jan 2022 15:52:11 +0100
Subject: [PATCH 0407/1695] plat-stm32mp2: conf: support HPDMA driver

Default enable HPDMA driver for platform stm32mp2.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 945c4b13d1b4bb7c33e35a677030d6f1f0d353cc plat-stm32mp2: conf: support HPDMA driver]
Signed-off-by: Gatien Chevallier <gatien.chevallier@st.com>
Change-Id: Ie7b00a3effddce66275aeddb614b875dbb9c2ab6
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/246291
Reviewed-by: Nicolas LE BAYON <nicolas.le.bayon@st.com>
Reviewed-by: Lionel DEBIEVE <lionel.debieve@foss.st.com>
---
 core/arch/arm/plat-stm32mp2/conf.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/plat-stm32mp2/conf.mk b/core/arch/arm/plat-stm32mp2/conf.mk
index f8677eaa7..710a26de6 100644
--- a/core/arch/arm/plat-stm32mp2/conf.mk
+++ b/core/arch/arm/plat-stm32mp2/conf.mk
@@ -70,6 +70,7 @@ CFG_STM32_BSEC3 ?= y
 CFG_STM32_EXTI ?= y
 CFG_STM32_FMC ?= y
 CFG_STM32_GPIO ?= y
+CFG_STM32_HPDMA ?= y
 CFG_STM32_IAC ?= y
 CFG_STM32_OMM ?= y
 CFG_STM32_RIF ?= y
-- 
2.47.2

